modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -6,41 +6,197 @@
 [id="ztp-configuring-disk-partitioning_{context}"]
 = Configuring disk partitioning with SiteConfig
 
-Configure disk partitioning for a managed cluster using a `SiteConfig` CR and GitOps ZTP. The  disk partition details in the `SiteConfig` CR must match the underlying disk.
+Configure disk partitioning for a managed cluster using a `SiteConfig` CR and {ztp-first}. The  disk partition details in the `SiteConfig` CR must match the underlying disk.
```

Comment:
This attribute is not rendering in the preview. It looks like the `{ztp-first}` attribute is missing from the `_attributes/common-attributes.adoc` file in `enterprise-4.12`. Can you add it in this PR?

Reply:
good catch added